content/300-guides/050-database/300-seed-database.mdx

```diff
@@ -40,6 +40,12 @@ Prisma's integrated seeding functionality expects a command in the `"seed"` key
 
 </TabbedContent>
 
+<Admonition type="info">
+
+When using containers with limited RAM, consider using the `--transpile-only` flag to avoid potential performance issues. The process of typechecking the seed script(s), in addition to transpiling them, can consume large amounts of RAM.
```

Comment:
I think an admonition addresses the issue nicely, my only suggestion is to show the actual code to use i.e. `"seed": "ts-node --transpile-only prisma/seed.ts"` somewhere rather than just showing the flag and leaving it to the user to figure out where/how to use it, sometimes flags can be tricky and not work if added in the wrong order.

Reply:
Sounds good 👍🏼